### ISSUE TITLE
Chucker 3.3.0 is incompatible with projects using Kotlin 1.4.20 and @Parcelize

### ISSUE THREAD

**Issue by ephemient** [ChuckerTeam/chucker#480]
## :writing_hand: Describe the bug
`@Parcelize` has moved out of the Kotlin Android Extensions (which is deprecated) into its own compiler plugin. It's using a different package name, but the new runtime still defines the old classes to provide deprecation errors. With both the old and new runtimes defining the same classes, this causes Android's `checkDebugDuplicateClasses` build task to fail.

## :bomb: Steps to reproduce

1. Create Android application using Kotlin 1.4.20-M2 (or whatever version is now available) and using `plugins { kotlin("plugin.parcelize") }`.
2. Add Chucker 3.3.0.
3. Try to build.
4. ```
    > Task :app:checkDebugDuplicateClasses FAILED

    FAILURE: Build failed with an exception.

    * What went wrong:
    Execution failed for task ':app:checkDebugDuplicateClasses'.
    > A failure occurred while executing com.android.build.gradle.internal.tasks.CheckDuplicatesRunnable
       > Duplicate class kotlinx.android.parcel.IgnoredOnParcel found in modules jetified-kotlin-android-extensions-runtime-1.4.10 (org.jetbrains.kotlin:kotlin-android-extensions-runtime:1.4.10) and jetified-kotlin-parcelize-runtime-1.4.20-M2 (org.jetbrains.kotlin:kotlin-parcelize-runtime:1.4.20-M2)
         Duplicate class kotlinx.android.parcel.Parceler found in modules jetified-kotlin-android-extensions-runtime-1.4.10 (org.jetbrains.kotlin:kotlin-android-extensions-runtime:1.4.10) and jetified-kotlin-parcelize-runtime-1.4.20-M2 (org.jetbrains.kotlin:kotlin-parcelize-runtime:1.4.20-M2)
         Duplicate class kotlinx.android.parcel.Parceler$DefaultImpls found in modules jetified-kotlin-android-extensions-runtime-1.4.10 (org.jetbrains.kotlin:kotlin-android-extensions-runtime:1.4.10) and jetified-kotlin-parcelize-runtime-1.4.20-M2 (org.jetbrains.kotlin:kotlin-parcelize-runtime:1.4.20-M2)
         Duplicate class kotlinx.android.parcel.Parcelize found in modules jetified-kotlin-android-extensions-runtime-1.4.10 (org.jetbrains.kotlin:kotlin-android-extensions-runtime:1.4.10) and jetified-kotlin-parcelize-runtime-1.4.20-M2 (org.jetbrains.kotlin:kotlin-parcelize-runtime:1.4.20-M2)
         Duplicate class kotlinx.android.parcel.RawValue found in modules jetified-kotlin-android-extensions-runtime-1.4.10 (org.jetbrains.kotlin:kotlin-android-extensions-runtime:1.4.10) and jetified-kotlin-parcelize-runtime-1.4.20-M2 (org.jetbrains.kotlin:kotlin-parcelize-runtime:1.4.20-M2)
         Duplicate class kotlinx.android.parcel.TypeParceler found in modules jetified-kotlin-android-extensions-runtime-1.4.10 (org.jetbrains.kotlin:kotlin-android-extensions-runtime:1.4.10) and jetified-kotlin-parcelize-runtime-1.4.20-M2 (org.jetbrains.kotlin:kotlin-parcelize-runtime:1.4.20-M2)
         Duplicate class kotlinx.android.parcel.WriteWith found in modules jetified-kotlin-android-extensions-runtime-1.4.10 (org.jetbrains.kotlin:kotlin-android-extensions-runtime:1.4.10) and jetified-kotlin-parcelize-runtime-1.4.20-M2 (org.jetbrains.kotlin:kotlin-parcelize-runtime:1.4.20-M2)

         Go to the documentation to learn how to <a href="d.android.com/r/tools/classpath-sync-errors">Fix dependency resolution errors</a>.
    ```


## :wrench: Expected behavior
Success

## :camera: Screenshots
<!-- If applicable, add screenshots to help explain your problem. -->

## :iphone: Tech info
 - Device: <!-- e.g. Nexus 6P -->
 - OS: <!-- e.g. 7.1.1 -->
 - Chucker version: <!-- e.g. 3.1.2 -->

## :page_facing_up: Additional context
<!-- Add any other context about the problem here. -->

**Comment by vbuberen** [comment]
Thanks for reporting the issue. 

I thought we removed `kotlin-android-extensions` plugin at all when migrated to `ViewBinding`. Seems I confused with other projects where I removed this plugin completetly.

We will fix it and release in `3.3.1` along with some other important fixes before final `1.4.20` Kotlin released.

**Comment by vbuberen** [comment]
@ephemient You can switch to the latest snapshot version for now, since we just merged the fix.
